tock/employees/models.py

```diff
@@ -130,6 +131,14 @@ def is_billable(self):
         """
         return self.billable_expectation > Decimal(0.0)
 
+    @cached_property
+    def billable_expectation(self):
```

Comment:
Interesting. In my head we'd have used a non-editable field, but i like this approach too!

Reply:
Yeah, could go either way here. I started with a non-editable field, but making it a property seemed sort of self-documenting, and we avoid storing redundant data (`billable_expectation` should always just be a function of `expected_billable_hours`). 🤷‍♂️